### PR TITLE
Side panel

### LINKED ISSUE #10: [TODO] Add pane to change options
roughjs has a bunch of options regarding line width, stroke/bg color, various patterns/roughness... We should add a pane where we can modify all those values for each element.

### PR DESCRIPTION
**Pull request by paulomenezes**
Fixes #10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
   "keywords": [],
   "main": "src/index.js",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.26",
```
Comment (vjeux) [inline package.json]: Do you need to import this, you don't seem to be using it.

File: src/index.tsx [hunk 2]
```diff
@@ -756,28 +765,111 @@ class App extends React.Component<{}, AppState> {
...
+              />
+              px)
+            </div>
+          </div>
+        </div>
         <canvas
           id="canvas"
-          width={window.innerWidth}
-          height={window.innerHeight - 210}
+          width={window.innerWidth - 250}
```
Comment (vjeux) [inline src/index.tsx]: Nice, now that it's a fixed width, it's the correct size :)

File: src/index.tsx [hunk 3]
```diff
@@ -598,28 +606,28 @@ const KEYS = {
...
     value: "rectangle"
   },
   {
-    label: "Ellipse",
+    icon: faCircle,
     value: "ellipse"
   },
   {
-    label: "Arrow",
+    icon: faExpandAlt,
```
Comment (vjeux) [inline src/index.tsx]: Is there a better icon for an arrow? That doesn't really look like an arrow

Comment (paulomenezes) [reply src/index.tsx]: The Zwibbler uses: https://fontawesome.com/icons/chart-line?style=solid but we have another arrows: https://fontawesome.com/icons?d=gallery&q=arrow&m=free

File: package.json (hunk 1 above)
Comment (vjeux) [reply package.json]: Maybe long-arrow-alt-right

File: src/index.tsx (hunk 3 above)
Comment (vjeux) [reply src/index.tsx]: Maybe long-arrow-alt-right

File: package.json (hunk 1 above)
Comment (paulomenezes) [reply package.json]: I think we need this for react-fontawesome. I got `Module not found: Can't resolve '@fortawesome/fontawesome-svg-core' in '/Users/paulomenezes/repositories/excalidraw/node_modules/@fortawesome/react-fontawesome'` when remove

Comment (vjeux) [reply package.json]: Sounds good :)

**Comment by vjeux** [comment]
Yay!

**Pull request merged**